target_core_user_local.h

```diff
@@ -4,7 +4,7 @@
 /* This header will be used by application too */
 
 #include <linux/types.h>
-#include <linux/uio.h>
+//#include <linux/uio.h>
 
```

Comment:
Drop this patch from the set. We will work on it so it works with old and new setups in the other PR.

Reply:
Sure, will do